src/HttpHeader.cc

```diff
@@ -1407,7 +1404,7 @@ HttpHeaderEntry::~HttpHeaderEntry()
 
 /* parses and inits header entry, returns true/false */
 HttpHeaderEntry *
-HttpHeaderEntry::parse(const char *field_start, const char *field_end)
+HttpHeaderEntry::parse(const char *field_start, const char *field_end, http_hdr_owner_type &msgType)
```

Comment:
The new API should not imply that parse() modifies msgType.

```suggestion
HttpHeaderEntry::parse(const char *field_start, const char *field_end, http_hdr_owner_type msgType)
```

or

```suggestion
HttpHeaderEntry::parse(const char *field_start, const char *field_end, const http_hdr_owner_type msgType)
```

Reply:
Done.